layer/mounted_layer.go

```diff
@@ -4,18 +4,21 @@ import (
 	"io"
 	"sync"
 
+	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/pkg/archive"
 )
 
 type mountedLayer struct {
-	name       string
-	mountID    string
-	initID     string
-	parent     *roLayer
-	path       string
-	layerStore *layerStore
-
-	references map[RWLayer]*referencedRWLayer
+	name         string
+	mountID      string
+	initID       string
+	parent       *roLayer
+	path         string
+	pathCache    string
+	layerStore   *layerStore
+	activeMount  int
```

Comment:
nit: use same names (activityCount) as `referenceLayer`. Mutex could be on the object but doesn't matter if `referenceLayer` already uses `activityL` name